src/ast/codegen_llvm.cpp

```diff
@@ -158,11 +158,8 @@ void CodegenLLVM::visit(Builtin &builtin)
       offset = arch::arg_offset(arg_num);
     }
 
-    AllocaInst *dst = b_.CreateAllocaBPF(builtin.type, builtin.ident);
-    Value *src = b_.CreateGEP(ctx_, b_.getInt64(offset * sizeof(uintptr_t)));
-    b_.CreateProbeRead(dst, builtin.type.size, src);
-    expr_ = b_.CreateLoad(dst);
-    b_.CreateLifetimeEnd(dst);
+    auto v = b_.getInt64(offset * sizeof(uintptr_t));
```

Comment:
Not sure if I'm being stupid, but what does `v` stand for here?

Reply:
Just a left over from refactoring. Ill merge the lines :)